tini/src/seccomp_fd_notify.c

```diff
@@ -277,12 +256,14 @@ static int install_notify_filter(void) {
 	/* Only one listening file descriptor can be established. An attempt to
 	   establish a second listener yields an EBUSY error. */
 
-	if (get_kernel_version(&system_kernel_version) == -1) {
-		return -1;
-	}
+	for (unsigned int i = 0; i < ARRAY_SIZE(flags); i++) {
+		seccomp(SECCOMP_SET_MODE_FILTER, flags[i], NULL);
```

Comment:
seems a little weird not to check the result of this, but I think it's probably ok since if this behavior changes that means a kernel ABI break, which the kernel's tests will check for.

Reply:
I agree. I wanted to check the return value as well, but the assumption is the seccomp call will return a -1 always if the arg field is NULL.

Reply:
> I wanted to check the return value as well, but the assumption is the seccomp call will return a -1 always if the arg field is NULL.

Sure, I guess I'm just saying i'd feel safer if we "trust-but-verified" :)